src/main/java/org/onedatashare/server/service/oauth/BoxOauthService.java

```diff
@@ -0,0 +1,85 @@
+package org.onedatashare.server.service.oauth;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.*;
+import org.onedatashare.server.model.core.Credential;
+import org.onedatashare.server.model.credential.OAuthCredential;
+import org.onedatashare.server.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+
+import java.util.Collection;
+
+/**
+ *author: Javier Falca
+ */
+
+@Service
+public class BoxOauthService {
+
+    @Autowired
+    private UserService userService;
+
+
+    public static class BoxConfig {
+
+        public static String client_id = "7xifcu329elepngu76j3k5vart312ibr";
+        public static String client_secret = "UPGc0WGVHTNB6UmYU2RPneXPA2yJB5Qp";
+        public static String redirect_uri = "http://127.0.0.1:8080/api/stork/oauth/box";
+        public static String box_redirect = "https://account.box.com/api/oauth2/authorize";
+        public static String scope = "root_readwrite";
+    }
```

Comment:
We store credentials/secrets in environment variables. Making it public is a security risk.

Reply:
I'll move them to environment variables